### ISSUE TITLE
Template indentation

### ISSUE THREAD

**Issue by hirokiosame** [vuejs/eslint-plugin-vue#194]
**Please describe what the rule should do:**
Enforce indentation type in the template: tab or spaces.

**What category of rule is this? (place an "X" next to just one item)**

[x] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```vue
<template>
    <div>
        <h2>Welcome</h2>
    </div>
</template>
```

**Comment by hirokiosame** [comment]
Sorry, I just saw there's already work on this. Great! Closing.
